libs/util/http.client.js

```diff
@@ -39,7 +39,9 @@ if (!String.prototype.trim) {
 }
 
 function normalizeHeaders(headers, method) {
-    var normalized = {};
+    var normalized = {
+        'X-Requested-With': 'XMLHttpRequest'
+    };
```

Comment:
`X-Requested-With` is a common header used for identify Ajax requests: http://en.wikipedia.org/wiki/List_of_HTTP_header_fields#Common_non-standard_request_fields